### PR TITLE
sample ai carecradle

### PR DESCRIPTION
**Pull request by Rajkumarpaneru18**

### CODE REVIEW

**Comment by Rajkumarpaneru18** [comment]
sample ai carecradle

**Comment by Rajkumarpaneru18** [comment]
sample ai carecradle

File: README.md [hunk 1]
```diff
@@ -1,34 +1,81 @@
...
+
+https://github-production-user-asset-6210df.s3.amazonaws.com/82999440/393319110-39fb1778-7c7a-4f0a-984b-2cf1ed5a5c74.mp4
+
+## 🚀 Getting Started
+
+### Open Using Daytona
+- **Install Daytona**:Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/). 
+- **Create the Workspace**:  
+  ```bash
+  daytona create https://github.com/Rajkumarpaneru18/CareCradle
```
Comment (Tpuljak) [inline README.md]: The link is wrong, should be https://github.com/daytonaio/sample-ai-carecradle

**Review by Tpuljak — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Rajkumarpaneru18) [reply README.md]: i did it please review it..

File: README.md [hunk 2]
```diff
@@ -1,34 +1,81 @@
...
+
+https://github-production-user-asset-6210df.s3.amazonaws.com/82999440/393319110-39fb1778-7c7a-4f0a-984b-2cf1ed5a5c74.mp4
+
+## 🚀 Getting Started
+
+### Open Using Daytona
+- **Install Daytona**:Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/). 
+- **Create the Workspace**:  
+  ```bash
+  https://github.com/daytonaio/sample-ai-carecradle
```
Comment (Tpuljak) [inline README.md]: Should be `daytona create https://github.com/daytonaio/sample-ai-carecradle`

**Review by Tpuljak — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Rajkumarpaneru18) [reply README.md]: ohh sorry,I did a mistake. I changed it. please review.

**Pull request merged**